### PR TITLE
Add p5p to helpers

### PR DESCRIPTION
**Pull request by scotthain**
@chef/engineering-services

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ AllCops:
     - mixlib-install.gemspec
 Style/AlignParameters:
   EnforcedStyle: with_first_parameter
+Style/BlockDelimiters:
```
Comment (wrightp) [inline .rubocop.yml]: curious. this project uses chef-style so the rubocop config should be removed altogether.

Comment (scotthain) [inline .rubocop.yml]: If I remove the .rubocop.yml I get oodles of `spec/mixlib/install/product_spec.rb:70:19: C: Style/BlockDelimiters: Avoid using {...} for multi-line blocks.
    let(:product) {` type things.... any suggestions? I can modify the code in all those places but that seems... wrong?

Comment (mwrock) [inline .rubocop.yml]: I think its the right thing but unrelated to the change in this PR. I hit this as well in #134 and have submitted #136 to fix this separately.

Comment (schisamo) [inline .rubocop.yml]: I asked @mwrock to remove the `rubocop.yml` file in https://github.com/chef/mixlib-install/pull/136.

**Comment by scotthain** [comment]
@chef/engineering-services for re-review please

**Comment by schisamo** [comment]
👍

**Comment by yzl** [comment]
👍

**Pull request merged**